gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -186,7 +186,7 @@ static String excludedPathsRegex() {
         // Error-prone normalizes filenames to use '/' path separator:
         // https://github.com/google/error-prone/blob/c601758e81723a8efc4671726b8363be7a306dce
         // /check_api/src/main/java/com/google/errorprone/util/ASTHelpers.java#L1277-L1285
-        return ".*/(build|generated_.*[sS]rc|src/generated.*)/.*";
+        return ".*/((build/.*)?generated_.*[sS]rc|src/generated.*)/.*";
```

Comment:
This isn't right - we shouldn't be dropping the alternation here.

We want to ignore directories like `generated_src`. That's what some our legacy [gradle-processors](https://github.com/palantir/gradle-processors) plugin uses and some internal repos have unfortunately adopted that as convention.

I propose we make this regex:
```
.*/(build/generated|src/generated|generated_.*[S]rc)/.*
```

Reply:
We have historical patterns using this format: `tritium-core/generated_src/` (sibling to the `build/` dir)